AdminWebsite/AdminWebsite/Extensions/HearingDetailsResponseExtensions.cs

```diff
@@ -90,7 +90,13 @@ private static OtherInformationDetails GetOtherInformationObject(string otherInf
             }
             catch (Exception)
             {
-                return new OtherInformationDetails {OtherInformation = otherInformation};
+                var properties = otherInformation.Split("|");
+                if (properties.Length > 1)
```

Comment:
should this be > 2

Reply:
We just care if the length is 2 or higher? Because we want the 3rd element.

Reply:
length doesnt start from 0 but index does